python/src/ops.cpp

```diff
@@ -3476,4 +3476,34 @@ void init_ops(py::module_& m) {
       Returns:
         result (array): The tiled array.
     )pbdoc");
+  m.def(
+      "addmm",
+      &addmm,
+      "c"_a,
+      "a"_a,
+      "b"_a,
+      py::pos_only(),
+      "alpha"_a = 1.0f,
+      "beta"_a = 1.0f,
+      py::kw_only(),
+      "stream"_a = none,
+      R"pbdoc(
+        addmm(c: array, a: array, b: array, /, alpha: float = 1.0, beta: float = 1.0,  *, stream: Union[None, Stream, Device] = None) -> array
+
+        Matrix multiplication and linear combination.
```

Comment:
I think `Matrix multiplication with addition.` is more clear.

Reply:
Should I make it `Matrix multiplication with addition and optional scaling` or am I overthinking ?

Reply:
Yea! Might as well!